<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QgsDecorationGridDialog</class>
 <widget class="QDialog" name="QgsDecorationGridDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>511</width>
    <height>353</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="sizeConstraint">
    <enum>QLayout::SetMinimumSize</enum>
   </property>
   <item>
    <layout class="QGridLayout" name="gridLayout_3">
     <item row="1" column="0">
      <widget class="QLabel" name="mGridTypeLabel">
       <property name="accessibleName">
        <string extracomment="Hello translotor"/>
       </property>
       <property name="text">
        <string>Grid type</string>
       </property>
       <property name="wordWrap">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="1" column="2">
      <widget class="QComboBox" name="mGridTypeComboBox"/>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="mIntervalXLabel">
       <property name="text">
        <string>Interval X</string>
       </property>
       <property name="wordWrap">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="2" column="2">
      <widget class="QDoubleSpinBox" name="mIntervalXSpinBox">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="decimals">
        <number>5</number>
       </property>
       <property name="maximum">
        <double>9999999.000000000000000</double>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="mIntervalYLabel">
       <property name="text">
        <string>Interval Y</string>
       </property>
       <property name="wordWrap">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="3" column="2">
      <widget class="QDoubleSpinBox" name="mIntervalYSpinBox">
       <property name="decimals">
        <number>5</number>
       </property>
       <property name="maximum">
        <double>9999999.000000000000000</double>
       </property>
      </widget>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="mOffsetXLabel">
       <property name="text">
        <string>Offset X</string>
       </property>
       <property name="wordWrap">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="4" column="2">
      <widget class="QDoubleSpinBox" name="mOffsetXSpinBox">
       <property name="decimals">
        <number>5</number>
       </property>
       <property name="maximum">
        <double>9999999.000000000000000</double>
       </property>
      </widget>
     </item>
     <item row="5" column="0">
      <widget class="QLabel" name="mOffsetYLabel">
       <property name="text">
        <string>Offset Y</string>
       </property>
       <property name="wordWrap">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="5" column="2">
      <widget class="QDoubleSpinBox" name="mOffsetYSpinBox">
       <property name="decimals">
        <number>5</number>
       </property>
       <property name="maximum">
        <double>9999999.000000000000000</double>
       </property>
      </widget>
     </item>
     <item row="6" column="0">
      <widget class="QLabel" name="mCrossWidthLabel">
       <property name="text">
        <string>Cross width</string>
       </property>
       <property name="wordWrap">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="6" column="2">
      <widget class="QDoubleSpinBox" name="mCrossWidthSpinBox">
       <property name="decimals">
        <number>5</number>
       </property>
      </widget>
     </item>
     <item row="7" column="0">
      <widget class="QLabel" name="mLineWidthLabel">
       <property name="text">
        <string>Line width</string>
       </property>
       <property name="wordWrap">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="7" column="2">
      <widget class="QDoubleSpinBox" name="mLineWidthSpinBox">
       <property name="decimals">
        <number>5</number>
       </property>
      </widget>
     </item>
     <item row="8" column="0">
      <widget class="QLabel" name="mLineColorLabel">
       <property name="text">
        <string>Line color</string>
       </property>
       <property name="wordWrap">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="8" column="2">
      <widget class="QgsColorButton" name="mLineColorButton">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="5" column="3">
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Fixed</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>10</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="0" column="0">
      <widget class="QCheckBox" name="chkEnable">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="layoutDirection">
        <enum>Qt::LeftToRight</enum>
       </property>
       <property name="text">
        <string>Enable grid</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="0" column="4" rowspan="9">
      <widget class="QGroupBox" name="mDrawAnnotationCheckBox">
       <property name="title">
        <string>Draw annotation</string>
       </property>
       <property name="flat">
        <bool>false</bool>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
       <property name="checked">
        <bool>false</bool>
       </property>
       <layout class="QGridLayout" name="gridLayout">
        <property name="sizeConstraint">
         <enum>QLayout::SetMinimumSize</enum>
        </property>
        <item row="0" column="0">
         <widget class="QLabel" name="mAnnotationPositionLabel">
          <property name="text">
           <string>Annotation position</string>
          </property>
          <property name="wordWrap">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QComboBox" name="mAnnotationPositionComboBox"/>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="mAnnotationDirectionLabel">
          <property name="frameShape">
           <enum>QFrame::NoFrame</enum>
          </property>
          <property name="text">
           <string>Annotation direction</string>
          </property>
          <property name="wordWrap">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QComboBox" name="mAnnotationDirectionComboBox"/>
        </item>
        <item row="2" column="0" colspan="2">
         <widget class="QPushButton" name="mAnnotationFontButton">
          <property name="text">
           <string>Font...</string>
          </property>
         </widget>
        </item>
        <item row="3" column="0">
         <widget class="QLabel" name="mDistanceToFrameLabel">
          <property name="text">
           <string>Distance to map frame</string>
          </property>
          <property name="wordWrap">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item row="3" column="1">
         <widget class="QDoubleSpinBox" name="mDistanceToMapFrameSpinBox"/>
        </item>
        <item row="4" column="0">
         <widget class="QLabel" name="mCoordinatePrecisionLabel">
          <property name="text">
           <string>Coordinate precision</string>
          </property>
          <property name="wordWrap">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item row="4" column="1">
         <widget class="QSpinBox" name="mCoordinatePrecisionSpinBox"/>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Apply|QDialogButtonBox::Cancel|QDialogButtonBox::Help|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QgsColorButton</class>
   <extends>QToolButton</extends>
   <header>qgscolorbutton.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
